src/kernel/arch/aarch64/arch.zig

```diff
@@ -29,7 +30,14 @@ extern var KERNEL_PHYSADDR_START: *u32;
 extern var KERNEL_PHYSADDR_END: *u32;
 
 pub fn initTTY(boot_payload: BootPayload) TTY {
-    return undefined;
+    tty.init(null, boot_payload);
```

Comment:
@SamTebbs33 What do I use for an allocator instead of null?

Reply:
I think it would make sense to pass an allocator to this function, but until that change is made to develop for x86, I think you can just use `mem.fixed_buffe_allocator` here.

Reply:
Thanks, that works.